File: Backend/VestTour.Domain/Entities/Measurement.cs
using System;
using System.Collections.Generic;

namespace VestTour.Domain.Entities;

public partial class Measurement
{
    public int MeasurementId { get; set; }

    public int? UserId { get; set; }

    public decimal? Weight { get; set; }

    public decimal? Height { get; set; }

    public decimal? Neck { get; set; }

    public decimal? Hip { get; set; }

    public decimal? Waist { get; set; }

    public decimal? Armhole { get; set; }

    public decimal? Biceps { get; set; }

    public decimal? PantsWaist { get; set; }

    public decimal? Crotch { get; set; }

    public decimal? Thigh { get; set; }

    public decimal? PantsLength { get; set; }

    public int? Age { get; set; }

    public decimal? Chest { get; set; }

    public decimal? Shoulder { get; set; }

    public decimal? SleeveLength { get; set; }

    public decimal? JacketLength { get; set; }

    public virtual ICollection<Product> Products { get; set; } = new List<Product>();

    public virtual User? User { get; set; }
}

File: Mobile/mobileapp/lib/models/measurement.dart
class Measurement {
  int? measurementId;
  int? userId;
  double? weight;
  double? height;
  double? neck;
  double? hip;
  double? waist;
  double? armhole;
  double? biceps;
  double? pantsWaist;
  double? crotch;
  double? thigh;
  double? pantsLength;
  dynamic age;
  dynamic chest;
  dynamic shoulder;
  dynamic sleeveLength;
  dynamic jacketLength;

  Measurement({
    this.measurementId,
    this.userId,
    this.weight,
    this.height,
    this.neck,
    this.hip,
    this.waist,
    this.armhole,
    this.biceps,
    this.pantsWaist,
    this.crotch,
    this.thigh,
    this.pantsLength,
    this.age,
    this.chest,
    this.shoulder,
    this.sleeveLength,
    this.jacketLength,
  });

  Measurement.fromJson(Map<String, dynamic> json) {
    measurementId = json["measurementId"];
    userId = json["userId"];
    weight = json["weight"]?.toDouble();
    height = json["height"]?.toDouble();
    neck = json["neck"]?.toDouble();
    hip = json["hip"]?.toDouble();
    waist = json["waist"]?.toDouble();
    armhole = json["armhole"]?.toDouble();
    biceps = json["biceps"]?.toDouble();
    pantsWaist = json["pantsWaist"]?.toDouble();
    crotch = json["crotch"]?.toDouble();
    thigh = json["thigh"]?.toDouble();
    pantsLength = json["pantsLength"]?.toDouble();
    age = json["age"];
    chest = json["chest"];
    shoulder = json["shoulder"];
    sleeveLength = json["sleeveLength"];
    jacketLength = json["jacketLength"];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    data["measurementId"] = measurementId;
    data["userId"] = userId;
    data["weight"] = weight;
    data["height"] = height;
    data["neck"] = neck;
    data["hip"] = hip;
    data["waist"] = waist;
    data["armhole"] = armhole;
    data["biceps"] = biceps;
    data["pantsWaist"] = pantsWaist;
    data["crotch"] = crotch;
    data["thigh"] = thigh;
    data["pantsLength"] = pantsLength;
    data["age"] = age;
    data["chest"] = chest;
    data["shoulder"] = shoulder;
    data["sleeveLength"] = sleeveLength;
    data["jacketLength"] = jacketLength;
    return data;
  }
}
